### ISSUE TITLE
Add support for Operator '*, /, and %'

### ISSUE THREAD

**Issue by kenjones-cisco** [mikefarah/yq#49]
Add support for [Operator '*, /, and %'](https://stedolan.github.io/jq/manual/#example15)

```
These infix operators behave as expected when given two numbers. Division by zero raises an error. x % y computes x modulo y.

Multiplying a string by a number produces the concatenation of that string that many times. "x" * 0 produces null.

Dividing a string by another splits the first using the second as separators.

Multiplying two objects will merge them recursively: this works like addition but if both objects contain a value for the same key, and the values are objects, the two are merged with the same strategy.
```

**Comment by sarimarton** [comment]
Is this implemented? `*` works, but `/` doesn't seem so, it says `Error: parsing expression: Lexer error: could not match text starting at 11:43 failing at 11:45.
	unmatched text: "/ "`

**Comment by teejaded** [comment]
It isn't working for me either.

```bash
❯ yq --version
yq (https://github.com/mikefarah/yq/) version v4.31.2

❯ echo '{"a": 2, "b":2}' | yq '.a / .b'
Error: 1:4: invalid input text "/ .b"

❯ echo '{"a": 2, "b":2}' | jq '.a / .b'
1
```

**Comment by mikefarah** [comment]
Yeah sorry this was never implemented - I closed a bunch of issues that weren't getting attention; happy to reopen!

**Comment by mikefarah** [comment]
Released in 4.32.1 - Thanks again @teejaded :)
